From: Markus Koschany <apo@debian.org>
Date: Mon, 1 Oct 2018 18:09:55 +0200
Subject: inputstreamctor

Upstream uses the older 1.6.1 version to bootstrap jflex. Since inputstreamctor
was removed in version 1.7.0 we have to make this adjustment.

Forwarded: not-needed
---
 src/main/jflex/LexScan.flex | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/main/jflex/LexScan.flex b/src/main/jflex/LexScan.flex
index 72ec62d..434b2d2 100644
--- a/src/main/jflex/LexScan.flex
+++ b/src/main/jflex/LexScan.flex
@@ -42,8 +42,6 @@ import jflex.unicode.UnicodeProperties;
 %state CTOR_ARG, REGEXP_CODEPOINT_SEQUENCE
 %state STRING_CODEPOINT_SEQUENCE, CHARCLASS_CODEPOINT
 
-%inputstreamctor false
-
 %cupdebug
 
 %{
